src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
```

Comment:
Consider adding constants instead of magic numbers to improve readability and maintainability of your code. This way, the numbers will have a clear meaning and can be changed in one place if needed.